R/bootstrap.R

```diff
@@ -690,6 +735,29 @@ tabPanelBody <- function(value, ..., icon = NULL) {
   tabPanel(title = NULL, ..., value = value, icon = icon)
 }
 
+#' @rdname tabPanel
+#' @export
+navItem <- function(...) {
+  item <- tags$li(class = "nav-item form-inline", ...)
+  class(item) <- c("shiny_nav_item", class(item))
+  item
```

Comment:
`.shiny-nav-itme` is only used to identify, but could leverage shiny at some point

```suggestion
  tags$li(class = "shiny-nav-item nav-item form-inline", ...)
```

Reply:
Also beware that BS5 dropped `.form-inline` https://getbootstrap.com/docs/5.0/migration/#forms-2